### PR TITLE
Adjustments to GRPCv2 based on feedback

### PR DESCRIPTION
**Pull request by shjortConcordium**
## Purpose

_Describe the purpose of the pull request, link to issue describing the problem, etc.

## Changes

- Fix/Update changelogs
- Adjustments to grpc transaction events, reject reasons and chain update types.

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [x] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

File: packages/common/src/GRPCTypeTranslation.ts [hunk 1]
```diff
@@ -763,18 +771,24 @@ function trRejectReason(
...
+                },
             };
         case 'invalidReceiveMethod':
             return {
                 tag: Tag.InvalidReceiveMethod,
-                contents: [
-                    unwrapValToHex(reason.invalidReceiveMethod.moduleRef),
-                    unwrap(reason.invalidReceiveMethod.receiveName?.value),
-                ],
+                contents: {
```
[the remark this answers is not in the document]
Comment (shjortConcordium) [reply packages/common/src/GRPCTypeTranslation.ts:784]: We can discuss this, but I thought they could still use the contents field, seeing that the other still have a contents field? :thinking:

File: packages/common/src/types/chainUpdate.ts [hunk 2]
```diff
@@ -141,14 +146,14 @@ export type AddIdentityProvider = IpInfo;
...
+    address: Base58String;
 }
 
 export interface ProtocolUpdateDetails {
     message: string;
     specificationUrl: string;
-    specificationHash: string;
-    specificationAuxiliaryData?: string;
+    specificationHash: HexString;
+    specificationAuxiliaryData: string;
```
[the remark this answers is not in the document]
Comment (shjortConcordium) [reply packages/common/src/types/chainUpdate.ts:156]: It should. :upside_down_face:

File: packages/common/src/types/transactionEvent.ts [hunk 3]
```diff
@@ -139,6 +139,12 @@ export interface ModuleDeployedEvent {
 
 // Account and transfer Events
 
+export interface AccountTransferredEvent {
+    tag: TransactionEventTag.Transferred;
+    amount: bigint;
```
[the remark this answers is not in the document]
Comment (shjortConcordium) [reply packages/common/src/types/transactionEvent.ts:144]: Yeah. I changed some additional bigint fields to use the Amount alias.

**Comment by rasmus-kirk** [comment]
Please change rejectReasonNumber to rejectReasonBakerId

**Review by rasmus-kirk — APPROVED** [review]

**Pull request merged**